src/js/components/Settings/UserSettings/PerformanceSettings.vue

```diff
@@ -25,7 +25,7 @@
 		<h3>
 			{{ t('polls', 'A poll with many options and voters can have a heavy impact on client performance.') }}
 			{{ t('polls', 'Set the amount of voting cells (options x participants) up to which all voting cells should be displayed.') }}
-			{{ t('polls', 'If this threshold gets tresspasses only the current participant will be displayed, to avoid a performance breakdown.') }}
+			{{ t('polls', 'If this threshold gets tresspassed, only the current participant will be displayed, to avoid a performance breakdown.') }}
```

Comment:
Yes correct, but there is another typo you adopted (incorrect double `s`): `trespassed`

Reply:
Right, let me fix it. heh.